sqlmesh/migrations/v0057_add_table_format.py

```diff
@@ -0,0 +1,5 @@
+"""Add table_format to the model top-level properties"""
```

Comment:
I think this is correct? The goal is to rewrite the snapshots because their fingerprints all change with the addition of the property

Reply:
I'm guessing we'll continue supporting the old way of things. Should we deprecate usage of `iceberg` in `storage_format` and emit a warning?

Reply:
> What about migrating existing snapshots that currently specify iceberg in storage_format?
Wouldn't that lead to a big diff on the next plan because the model files wont have `table_format iceberg` in them?

> Should we deprecate usage of iceberg in storage_format and emit a warning?
Yes, that's a good idea

Reply:
I've added a warning in the model `after` validator